### ISSUE TITLE
[office-addin-lint] support TypeScript 3.8 syntax

### ISSUE THREAD

**Issue by clavx** [OfficeDev/Office-Addin-Scripts#280]
The new syntax in TypeScript 3.8 is not supported.
Could you update the dependent packages?

``` typescript
import type { IObjectWithKey } from "@uifabric/utilities";
```
```
  1:14  error  Parsing error: '=' expected  prettier/prettier
```
Thank you.

**Comment by LouMM** [comment]
Thank you for reporting the issue @clavx . 
We were work on a fix as part of our summer release.

**Comment by LouMM** [comment]
Re-assigning. Update Prettier and Typescript

**Comment by millerds** [comment]
Form what I can see this has already been updated.  @clavx, can you confirm if this is still a problem for you?

**Comment by clavx** [comment]
Thank you for your response.

I've confirmed that it works fine.

I think this issue was resolved by upgrading the eslint version from "6.8.0" to "7.20.0" by the https://github.com/OfficeDev/Office-Addin-Scripts/pull/388.
